src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -22,14 +22,18 @@ const ItemsContainer = styled.div`
 export const FilterSidebar = () => {
   const collectionFilters = useWalletCollections((state) => state.collectionFilters)
   const setCollectionFilters = useWalletCollections((state) => state.setCollectionFilters)
-
+  const isBagExpanded = useBag((state) => state.bagExpanded)
   const walletCollections = useWalletCollections((state) => state.walletCollections)
 
   const [isFiltersExpanded, setFiltersExpanded] = useFiltersExpanded()
   const isMobile = useIsMobile()
 
   const { sidebarX } = useSpring({
     sidebarX: isFiltersExpanded ? 0 : -360,
+    config: {
+      duration: 250,
```

Comment:
does it make sense to use durations from theme?

Reply:
this will require some refactoring as the durations give the full ms whereas the config requires a number. I've come up with an intermediary solution that uses an enum for duration values